include/tvm/runtime/profiling.h

```diff
@@ -205,6 +205,28 @@ class ReportNode : public Object {
    *  `aggregate` is true.
    */
   String AsTable(bool sort = true, bool aggregate = true) const;
+  /*! \brief Convert this report to json.
+   *
+   * Output json will be of this format:
```

Comment:
My eyes :-) Would you mind pretty printing this JSON in the comment? It makes the format more readable.

Reply:
Also, recommend using `JSON` (not `json`) in the comments for the sake of the autogenerated documentation.